test/unit/api/regular/shuffle_v2/shuffle_v2_test.hpp

```diff
@@ -47,14 +47,22 @@ verify(T x, eve::fixed<G>, eve::pattern_t<I...> p, U shuffled)
     return false;
   }();
 
-  TTS_EXPECT(!has_failures) << "sizeof(T): " << sizeof(T) << " G: " << G << "\npattern: " << p
-                            << "\nactual:  " << shuffled;
+  TTS_EXPECT(!has_failures) << "sizeof(T): " << sizeof(T) << " G: " << G
+                                      << "\npattern: " << p << "\nactual:  " << shuffled;
 }
 
 template<typename T, std::ptrdiff_t... I>
 void
 run_one_case(std::ptrdiff_t expected_level, T input, auto g, eve::pattern_t<I...> p)
 {
+#if 0
```

Comment:
can we delete that if not needed ? Can be done next patch

Reply:
Oh yeah - good catch.